### PR TITLE
Updated OutputFormat to match Multiview4 capabilities

### LINKED ISSUE #5: Unable to change output Video Format setting
I'm trying to get this module to change the Video Format (output) of the Multiview4

The valid configuration stanza's are below and each work if sent over telnet connection to the BMD Multiview4

CONFIGURATION:
Output format: 1080i50

CONFIGURATION:
Output format: 1080i5994

CONFIGURATION:
Output format: 2160p25

CONFIGURATION:
Output format: 2160p2997

I can't figure out from the code how I would go about changing / updating this module to support these setting as the current setting (50i/50p/60i/60p) don't seem to do anything.

I'm happy to hack away at the code if someone can point me in the right direction - is it as simple as changing the this.CHOICES_OUTPUTFORMAT array in index.js?  if so, which field is what is actually sent to the device?

### PR DESCRIPTION
**Pull request by bjeans**
Fixes #5

### CODE REVIEW

**Comment by krocheck** [comment]
@bjeans is this change tested?

**Comment by bjeans** [comment]
@krocheck I have been using this changed code in my environment since I implemented the fix.  Is there a formal process to request others to test?

**Comment by haakonnessjoen** [comment]
Are the old modes deprecated, or maybe still in use in an older model? Maybe we should let the old modes be?

**Comment by peternewman** [comment]
On the unit I've got running firmware v2.2.5 and PROTOCOL PREAMBLE Version: 2.8, it will ACK for example:
```
CONFIGURATION:
Output format: 50i
```

But the format doesn't actually change.

I guess as @haakonnessjoen says, they may still exist on older devices that haven't been plugged into the config tool since.

File: index.js [hunk 1]
```diff
@@ -61,10 +61,10 @@ class instance extends instance_skel {
 		];
 
 		this.CHOICES_OUTPUTFORMAT = [
-			{ id: '50i', label: '50i',  preset: '50i' },
-			{ id: '50p', label: '50p',  preset: '50p' },
-			{ id: '60i', label: '60i',  preset: '60i' },
-			{ id: '60p', label: '60p',  preset: '60p' }
+			{ id: '1080i50', label: '1080i50',  preset: '1080i50' },
+                        { id: '1080i5994', label: '1080i5994',  preset: '1080i5994' },
```
Comment (peternewman) [inline index.js]:
```suggestion
                        { id: '1080i5994', label: '1080i59.94',  preset: '1080i59.94' },
```

File: index.js [hunk 2]
```diff
@@ -61,10 +61,10 @@ class instance extends instance_skel {
...
+                        { id: '2160p25', label: '2160p25',  preset: '2160p25' },
+                        { id: '2160p2997', label: '2160p2997',  preset: '2160p2997' }
```
Comment (peternewman) [inline index.js]:
```suggestion
                        { id: '2160p2997', label: '2160p29.97',  preset: '2160p29.97' }
```

**Review by peternewman — COMMENTED** [review]
I think these would make it look slightly nicer

File: index.js [hunk 3]
```diff
@@ -60,11 +60,11 @@ class instance extends instance_skel {
...
 		this.CHOICES_OUTPUTFORMAT = [
-			{ id: '50i', label: '50i', preset: '50i' },
-			{ id: '50p', label: '50p', preset: '50p' },
-			{ id: '60i', label: '60i', preset: '60i' },
-			{ id: '60p', label: '60p', preset: '60p' },
-		]
+			{ id: '1080i50', label: '1080i50',  preset: '1080i50' },
+			{ id: '1080i5994', label: '1080i5994',  preset: '1080i5994' },
+			{ id: '2160p25', label: '2160p25',  preset: '2160p25' },
+			{ id: '2160p2997', label: '2160p2997',  preset: '2160p2997' }
```
Comment (peternewman) [inline index.js]: This keeps the old values for backwards compatibility (older units) and combines my optimisations from the earlier suggestion.
```suggestion
			{ id: '50i', label: '50i', preset: '50i' },
			{ id: '50p', label: '50p', preset: '50p' },
			{ id: '60i', label: '60i', preset: '60i' },
			{ id: '60p', label: '60p', preset: '60p' },
			{ id: '1080i50', label: '1080i50',  preset: '1080i50' },
			{ id: '1080i5994', label: '1080i59.94',  preset: '1080i59.94' },
			{ id: '2160p25', label: '2160p25',  preset: '2160p25' },
			{ id: '2160p2997', label: '2160p29.97',  preset: '2160p29.97' },
```

File: index.js (hunk 2 above)
Comment (peternewman) [reply index.js]: Now covered in https://github.com/bitfocus/companion-module-bmd-multiview4/pull/6#discussion_r1527170486

File: index.js (hunk 1 above)
Comment (peternewman) [reply index.js]: Now covered in https://github.com/bitfocus/companion-module-bmd-multiview4/pull/6#discussion_r1527170486

**Review by peternewman — APPROVED** [review]
LGTM thanks @bjeans .

**Pull request merged**